### DISCUSSION TITLE
Adding Persian Language to blacksheep documentation

### DISCUSSION THREAD

**Discussion by Legopapurida** [Neoteroi/BlackSheep#146] [Ideas]
### I am Parsa from IRan
I would be happy that you translate persian lang 
coz I wanna to publish and start a course for learning blacksheep,
if you need help to this I can collaborate with you to doing that.

Good luck man.

**Comment by myusko** [comment]
Hi @Legopapurida 

Right now, the translation of the documentation isn't on our to-do list, but any contributions regarding the documentation are welcome!

Thanks,
Mykhailo

**Reply by RobertoPrevato** [reply]
Hi @Legopapurida 
To add something more to what Mykhailo wrote, some work needs to be done on the documentation source code and CI/CD pipelines to support more languages.

Currently the documentation is in a project in Azure DevOps, soon we will migrate to a GitHub repository and GitHub Actions, and possibly the structure of the documentation's source code and the GitHub Actions will be made to support multi-language out of the box.

At that point, it will be easy to accept PRs to have documentation in more languages.
